docs/src/reference/mdx/atoms/Field.docs.mdx

```diff
@@ -69,6 +69,21 @@ import { Field } from '@ensdomains/thorin'
 </Stack>
 ```
 
+## Label Placement
+
+```tsx live=true
+<Stack>
```

Comment:
use DeleteMe

Reply:
I'm wondering what caused this to get back into the repository... I'm sure I removed all the stack before...